library/core/src/num/f32.rs

```diff
@@ -1008,29 +1008,37 @@ impl f32 {
         Self::from_bits(u32::from_ne_bytes(bytes))
     }
 
-    /// Returns an ordering between self and other values.
+    /// Return the ordering between `self` and `other`.
+    ///
     /// Unlike the standard partial comparison between floating point numbers,
     /// this comparison always produces an ordering in accordance to
-    /// the totalOrder predicate as defined in IEEE 754 (2008 revision)
-    /// floating point standard. The values are ordered in following order:
-    /// - Negative quiet NaN
-    /// - Negative signaling NaN
-    /// - Negative infinity
-    /// - Negative numbers
-    /// - Negative subnormal numbers
-    /// - Negative zero
-    /// - Positive zero
-    /// - Positive subnormal numbers
-    /// - Positive numbers
-    /// - Positive infinity
-    /// - Positive signaling NaN
-    /// - Positive quiet NaN
-    ///
-    /// Note that this function does not always agree with the [`PartialOrd`]
-    /// and [`PartialEq`] implementations of `f32`. In particular, they regard
-    /// negative and positive zero as equal, while `total_cmp` doesn't.
+    /// the `totalOrder` predicate as defined in the IEEE 754 (2008 revision)
+    /// floating point standard. The values are ordered in the following sequence:
+    ///
+    /// - negative quiet NaN
+    /// - negative signaling NaN
+    /// - negative infinity
+    /// - negative numbers
+    /// - negative subnormal numbers
+    /// - negative zero
+    /// - positive zero
+    /// - positive subnormal numbers
+    /// - positive numbers
+    /// - positive infinity
+    /// - positive signaling NaN
+    /// - positive quiet NaN.
```

Comment:
Not trying to nitpick, but I still don't think this list needs to act like a sentence.

```suggestion
    /// - positive quiet NaN
```

Reply:
I went around the internet looking for style guides maintained by the big-shots of all things English. Filtering out those requiring all sorts of punctuation between list items left me with the [style guide maintained by Oxford](https://www.ox.ac.uk/sites/files/oxford/media_wysiwyg/University%20of%20Oxford%20Style%20Guide.pdf). It specifies that the last bullet item should end with a full stop if the list forms a full sentence with the preceding text, which in this case it does (I think…?)

Personally I can live without a semicolons, but lack of a full stop in this situation is pretty jarring.

Reply:
@nagisa I think "if the list forms a full sentence" is the operative bit there, and I don't personally think it needs to.

However, I *also* really don't care enough about this, and you very clearly do, so please consider this not a blocker and r=me without it if you prefer.